pkg/fqdn/bootstrap/fqdn_bootstraper.go

```diff
@@ -44,18 +50,37 @@ const (
 	metricErrorAllow   = "allow"
 )
 
+type FQDNProxyBootstrapper interface {
+	BootstrapFQDN(possibleEndpoints map[uint16]*endpoint.Endpoint, preCachePath string) error
+	UpdateDNSDatapathRules(ctx context.Context) error
+}
+
+type fqdnProxyBootstrapper struct {
+	ctx               context.Context
+	logger            *slog.Logger
+	dnsNameManager    namemanager.NameManager
```

Comment:
nit: now that this is contained in the FQDN pkg, I think this can just be the `nameManager`

Reply:
agree, i renamed some others as well - given that the context is "DNS"